test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -292,15 +291,15 @@ private Map<String, Object> getBookkeeperImageSpec(String imageName) {
                 .build()).build();
     }
 
-    private Map<String, Object> getResources(String limitsCpu, String limitsMem, String requestsCpu, String requestsMem) {
+    private Map<String, Object> getResources(Resources resources) {
         return ImmutableMap.<String, Object>builder()
                 .put("limits", ImmutableMap.builder()
-                        .put("cpu", limitsCpu)
-                        .put("memory", limitsMem)
+                        .put("cpu", resources.getLimits().get("cpu"))
+                        .put("memory", resources.getLimits().get("memory"))
                         .build())
                 .put("requests", ImmutableMap.builder()
-                        .put("cpu", requestsCpu)
-                        .put("memory", requestsMem)
+                        .put("cpu", resources.getRequests().get("cpu"))
```

Comment:
can choose to introduce static constants here and at other places.

Reply:
There are not more than 2 instances of resources.getRequests or resourceWrapper.getControllerProperties or any of the other properties. So, even then do we need to instroduce static constants for each of them?